### PR TITLE
Go version update 1.13.4

### PR DESCRIPTION
**Pull request by mumutyal**

### CODE REVIEW

File: block-storage-attacher/utils/config/config.go [hunk 1]
```diff
@@ -71,7 +71,7 @@ func GetConfigStringList(envKey string, defaultConf string, logger zap.Logger) [
 }
 
 type Volume struct {
-	VolId    string `json:"iqn,omitempty"`
+	VolId    string `json:"volid,omitempty"`
```
Comment (akgunjal) [inline block-storage-attacher/utils/config/config.go]: I think this should be `volID`. Please check and update.

File: block-storage-attacher/helm/ibm-block-storage-attacher/RELEASENOTES.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ Fix for detach PV impacting other attacher pods
 - Changes to pv_watcher to ignore detach requests of other nodes.
 
 ## [v1.1.1] - 2019-08-28
-Update the GO lang version of block attacher image to 1.12.9
+Update the GO lang version of block attacher image to 1.13.4
```
[the remark this answers is not in the document]
Comment (mumutyal) [reply block-storage-attacher/helm/ibm-block-storage-attacher/RELEASENOTES.md]: Done

File: block-storage-attacher/utils/config/config.go (hunk 1 above)
Comment (mumutyal) [reply block-storage-attacher/utils/config/config.go]: Addressed the comment

Comment (akgunjal) [reply block-storage-attacher/utils/config/config.go]: I think its not addressed. Please check https://github.com/IBM/ibmcloud-storage-utilities/blob/8b058e6c78f20670359d600437cad511dd60d48f/block-storage-attacher/watcher/pv_watcher.go#L29

You need to match the case as well.

**Pull request merged**